client-python/pycti/api/opencti_api_client.py

```
@@ -91,9 +91,13 @@
 _PROXY_SIGNAL_HANDLERS_REGISTERED = False
 
 
-def build_request_headers(token: str, custom_headers: str, app_logger):
+def build_request_headers(token: str, custom_headers: str, app_logger, provider: str):
```

we should rather cumulate the information in the user agent. eg
* no additional parameter -> `pycti/6.8.15` (as today)
* with additional parameter `client/1.2.3` ->   `pycti/6.8.15 client/1.2.3`

---

we need to either validate the additional parameter or split in two part (name and version, but still need to be validated ?)

---

I added validation, worker currently doesn't have any version number associated though, so it defaults to a hardcoded value of 1.0